pkg/node/node.go

```diff
@@ -223,3 +223,31 @@ func (n *Node) ClusterDNS() string {
 	}
 	return "10.100.0.10"
 }
+
+const (
+	Arm64 = "arm64"
+	Amd64 = "x86_64"
+)
+
+func (n *Node) PauseImage() string {
+	var account, arch string
+	switch n.Region {
+	case "ap-east-1":
+		account = "800184023465"
+	case "me-south-1":
+		account = "558608220178"
+	default:
+		account = "602401143452"
+	}
+
+	switch *n.Architecture {
+	case Amd64:
+		arch = "amd64"
+	case Arm64:
+		arch = "arm64"
+	default:
+		panic(fmt.Sprintf("%s is not a supported machine architecture", *n.Architecture))
```

Comment:
This does get the job done but maybe it should be implemented as an Error type (https://blog.golang.org/go1.13-errors) and let the main execution flow (here: https://github.com/errm/ekstrap/blob/pause-enhancements/pkg/system/system.go#L66 ) decide whether to panic or not based on the error? Maybe it's too much complexity just for clarity and idiom's sake...

Reply:
I refactored ... and I just defaulted to `amd64` if the field is not set or is something unexpected!